x-pack/plugins/observability_solution/observability/public/components/tags.tsx

```diff
@@ -10,38 +10,53 @@ import React, { useState } from 'react';
 import { EuiBadge, EuiPopover } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 
-export function Tags({ tags }: { tags: string[] }) {
+export function Tags({
+  tags,
+  color,
+  size = 3,
+  oneLine = false,
+}: {
+  tags: string[];
+  color?: string;
+  size?: number;
+  oneLine?: boolean;
```

Comment:
there is same kinda component exported from observability-shared, might be nice to use that 

https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability_shared/public/index.ts#L38

Reply:
Nice! I will check it out!